features/lorawan/lorastack/phy/LoRaPHY.h

```diff
@@ -44,102 +44,163 @@ class LoRaPHY {
     LoRaPHY(LoRaWANTimeHandler &lora_time);
     virtual ~LoRaPHY();
 
+    /** Stores a reference to Radio object.
+     *
+     * Application is responsible for constructing a 'LoRaRadio' object
+     * which is passed down to the PHY layer.
+     *
+     * @param radio    a reference to radio driver object
+     */
     void set_radio_instance(LoRaRadio& radio);
 
+    /** Puts radio in sleep mode.
+     *
+     * Requests the radio driver to enter sleep mode.
+     */
     void put_radio_to_sleep(void);
 
+    /** Puts radio in standby mode.
+     *
+     * Requests the radio driver to enter standby mode.
+     */
     void put_radio_to_standby(void);
 
+    /** Puts radio in receive mode.
+     *
+     * Requests the radio driver to enter receive mode for a given time or to
+     * enter continuous reception mode.
+     *
+     * @param is_rx_continuous    if true, sets the radio to enter continuous
+     *                            reception mode.
+     *
+     * @param max_rx_window       duration of receive window
+     */
     void setup_rx_window(bool is_rx_continuous, uint32_t max_rx_window);
```

Comment:
This method could be refactored a bit. Change it to:
void LoRaPHY::setup_rx_window(bool rx_continuous, uint32_t max_rx_window) {
    _radio->lock();
    if (!rx_continuous) {
        _radio->receive(max_rx_window);
    } else {
       _radio->receive(0); // Continuous mode
    }
    _radio->unlock();
}